src/main/resources/xml/schema/2020-1/nordic-html5.rng

```diff
@@ -2289,6 +2289,7 @@
                             <value>line</value>
                             <value>line_indent</value>
                             <value>line_longindent</value>
+                            <value>sentence</value>
```

Comment:
This would allow sentence without line. Could you move it to a separate `<optional>`?

Reply:
Correct, that would be optional.